showOutput3.js

```diff
@@ -935,19 +769,81 @@ document.getElementById('submit').addEventListener(
             "};"
         );
 
-
         /**************
         ** info.json **
         **************/
         var infojsonOutput = [
             "{",
             "    \"keyboard_name\": \""+ obj.keyboard.settings.name +"\",",
+            "    \"manufacturer\": \"Your Name Here\",",
```

Comment:
How do you feel about leaving this as an empty string? KBF JSON exports don't have a field for this, so in the dev branch I did last year I had this blank, because it forces the user to set something manually to get the code to compiling.

Reply:
Yeah, was wondering what to do with this. Either way, should be easy to catch in review.